pkg/bundle/resources.go

```diff
@@ -34,7 +35,7 @@ func readResources(ctx context.Context, spec *fleet.BundleSpec, compress bool, b
 		return nil, err
 	}
 
-	var chartDirs []*fleet.HelmOptions
+	var chartDirs, parsedChartDirs []*fleet.HelmOptions
```

Comment:
Why do you need extra `parsedChartDirs`? Can we just use chartDirs, but overriding chartDir.Helm.Values? since you are already modifying/overriding the helm.Values.

Reply:
tweaked based on feedback.